File: src/main/java/ru/develgame/jsfgame/player/PlayerBean.java
package ru.develgame.jsfgame.player;

import ru.develgame.jsfgame.domain.Person;

import javax.enterprise.context.SessionScoped;
import javax.inject.Inject;
import java.io.Serializable;

@SessionScoped
public class PlayerBean implements Serializable {
    @Inject
    private Person person;

    private static final long serialVersionUID = -4473885537493656995L;

    public Person getPerson() {
        return person;
    }
}

File: src/main/kotlin/ru/develgame/jsfgame/player/PlayerBean.kt
package ru.develgame.jsfgame.player

import ru.develgame.jsfgame.domain.Person
import java.io.Serializable
import javax.enterprise.context.SessionScoped
import javax.inject.Inject

@SessionScoped
open class PlayerBean: Serializable {
    @Inject
    private lateinit var person: Person

    open fun getPerson() = person
}